src/Sylius/Bundle/AdminBundle/Resources/views/Security/Login/Content/_form.html.twig

```diff
@@ -0,0 +1,7 @@
+{% form_theme form 'bootstrap_5_layout.html.twig' %}
+{% set event_name = _context.event_prefix ~ '.form' %}
+
+{{ form_start(form, {'action': path('sylius_admin_login_check'), 'attr': {'class': 'ui large loadable form', 'novalidate': 'novalidate'}}) }}
+    {{ sylius_template_event(event_name, _context|merge({ event_prefix: event_name })) }}
```

Comment:
This action repeats several times `_context|merge({event_prefix: event_name})`, might be worth handling at `sylius_template_event` side or via `macro`